### ISSUE TITLE
Can't import new language

### ISSUE THREAD

**Issue by Kerblif** [OpenOLAT/OpenOLAT#105]
Hi!

For some reason, when importing the jar file, it says that everything was applied successfully, but I can't find the language in the list. Where can I find him? Well, or to figure out exactly what went wrong.

**Comment by gnaegi** [comment]
How did you create the jar file? Best is to export a language and then compare your jar with the exported. Make sure it has the exact same structure.

**Comment by Kerblif** [comment]
Yes, I checked that it looks the same as the exported one. I attached it.
[pack.jar.zip](https://github.com/user-attachments/files/16218666/pack.jar.zip)

**Comment by gnaegi** [comment]
Hi 
I don't get it, why do you want to import a language where every value is the same as the key? e.g. the name of the language: 

```
this.language.translated=this.language.translated
this.language.translated=this.language.translated
```

This does not make any sense at all. The idea is that you have the translation key on the left side and the value translated in your language on the right side. If you don't want to translate, don't add the key so the system will use the EN translation as default.

**Comment by Kerblif** [comment]
Hi!
Firstly, using such a "language" it is convenient to see where the text is in the interface.
Secondly, are there any restrictions on what a language can look like?
